nums/numpy/api.py

```diff
@@ -128,20 +141,22 @@ def array(object, dtype=None, copy=True, order="K", ndmin=0, subok=False) -> Blo
     return app.array(result, block_shape)
 
 
-def empty(shape, dtype=np.float):
+def empty(shape, dtype=float):
     app = _instance()
     block_shape = app.compute_block_shape(shape, dtype)
     return app.empty(shape=shape, block_shape=block_shape, dtype=dtype)
 
 
-def zeros(shape, dtype=np.float):
+def zeros(shape, dtype=float):
     app = _instance()
     block_shape = app.get_block_shape(shape, dtype)
     return app.zeros(shape=shape, block_shape=block_shape, dtype=dtype)
 
 
-def ones(shape, dtype=np.float):
+def ones(shape, dtype=float):
     app = _instance()
+    if isinstance(shape, int):
```

Comment:
Is integer shape supported by numpy? If so, can we move this somewhere where it will be triggered for other creation functions (e.g. empty and zeros)?

Reply:
Yes, it is supported. If I'm understanding your request correctly, you'd like to add this to zeros and empty?

Reply:
Like we discussed offline, I'll add a quick fix for `empty`, `zeros`, and `ones`, and look into a better fix in the future.